sdk/messaging/azeventhubs/go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventhubs
 go 1.18
 
 require (
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.4.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.8.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.1.0
-	github.com/Azure/azure-sdk-for-go/sdk/internal v1.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/internal v1.4.0
```

Comment:
Will need to revert to `v1.3.0` until Ben gets the tests updated.

Reply:
Made the fix to address this.  See updated change